RELEASE.md

```diff
@@ -0,0 +1,5 @@
+Release type: patch
+
+This release adds a `set_cookie` method to the `TemporalResponse` class so that the integrations Sanic, Django Channels and Chalice have a method for setting cookies.
```

Comment:
this is only for integration that don't support this already (via their response), right?

I wonder if we should remove our temporal response and use the standard response from all the integration, what do you think?

Reply:
I'm mostly thinking about consistency 😊

Reply:
I would really like removing the `TemporalResponse` class and I looked at the integrations:

1. Sanic has a [`Response`](https://github.com/sanic-org/sanic/blob/main/sanic/response/types.py#L244) class that supports adding / removing cookies, so using that is no problem.
2. Chalice has a [`Response`](https://github.com/aws/chalice/blob/79838b02dc330cfe549823899d2662ded0538015/chalice/app.py#L468) class but that doesn't support cookies. [Tests of that framework just set cookies via headers directly](https://github.com/aws/chalice/blob/79838b02dc330cfe549823899d2662ded0538015/tests/unit/test_local.py#L176).
3. Django Channels has no concept of a `Response` class. A consumer has to implement the [`handle`](https://github.com/django/channels/blob/2db92ccb57ff75b8f7535c3bc818f47d43a1b8bf/channels/generic/http.py#L55) method. The Strawberry implementation of that method uses its own [`ChannelsResponse`](https://github.com/strawberry-graphql/strawberry/blob/5503591baa28c5cea5ee2303f3fa00ca4a30f693/strawberry/channels/handlers/http_handler.py#L173) class.

So maybe this is a good idea moving forward:

1. Sanic: Use its own `Response` class.
2. Channels: Use Strawberry's `ChannelsResponse` and extend it with a `set_cookie` method. For that, I would extract the `set_cookie` method in this PR into a reuseable function.
3. Chalice: Use its `Response` class and create a small helper function (something like `set_cookie(response, value, ...)`) that adds the correct header to the `response` with the help of the function from 2.

What do you think? I can implement that. If you are OK with this idea, I would create a new PR for point 1 and reuse this PR for point 2. At a later time (after the PR for point 2 is merged), I would create a new PR for point 3.